### PR TITLE
Created subfolders in Active Task folder, created new component for my active request, added new routes to react router

### PR DESCRIPTION
**Pull request by JimmyG37**
lemon

### CODE REVIEW

File: client/src/components/ActiveTask/MyActive/MyActiveRequest.jsx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        <Grid
+          direction="column"
+          justifyContent="center"
+          alignItems="flex-start"
+          style={{ fontStyle: 'Roboto' }}
+        >
+          <YourHelper />
+          <ActiveModal />
+          <CancelButton>
+            <Button style={{ backgroundColor: '#EEEEEE', borderRadius: '24px', height: '50px', width: '200px', borderColor: '#EEEEEE', color: '#6C6C6C' }} >Cancel Request</Button>
```
Comment (raphaelspies) [inline client/src/components/ActiveTask/MyActive/MyActiveRequest.jsx:95]: Either put all styles on separate lines or create a new styled component

File: client/src/components/ActiveTask/YouAreHelping/Active.jsx [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+        <Grid
+          direction="column"
+          justifyContent="center"
+          alignItems="flex-start"
+          style={{ fontStyle: 'Roboto' }}
+        >
+          <YouAreHelping />
+          <MarkFinishModal />
+          <CancelButton>
+            <Button style={{ backgroundColor: '#EEEEEE', borderRadius: '24px', height: '50px', width: '200px', borderColor: '#EEEEEE', color: '#6C6C6C' }} >Cancel Request</Button>
```
Comment (raphaelspies) [inline client/src/components/ActiveTask/YouAreHelping/Active.jsx:93]: Either put all styles on separate lines or create a new styled component

File: client/src/components/ActiveTask/YouAreHelping/MarkFinishModal.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+  const handleClose = () => setAppear(false);
+  const handleShow = () => setAppear(true);
+  const handleFinish = (e) => setTaskFinish(e.target.value);
+
+  console.log(taskFinish);
+
+  return (
+    <div>
+      <MarkFinishButton>
+        <Button onClick={handleShow} style={{ backgroundColor: "#D65454", borderRadius: '24px', height: '50px', width: '200px', borderColor: '#D65454' }}>Mark as finished!</Button>
```
Comment (raphaelspies) [inline client/src/components/ActiveTask/YouAreHelping/MarkFinishModal.jsx:29]: Either put all styles on separate lines or create a new styled component

File: client/src/components/Helpful/HelpfulFeed.jsx [hunk 4]
```diff
@@ -12,28 +12,34 @@ const SideMenu = styled.div`
...
 `;
 
+// const MostHelpful = styled.div`
+// margin-left: 3em;
+// `;
+
 const HelpfulFeed = () => {
   const tasks = useSelector((store) => store.tasksReducer.tasks);
-
+  console.log('Tasssssskkksss', tasks);
```
Comment (raphaelspies) [inline client/src/components/Helpful/HelpfulFeed.jsx:21]: remove console log

**Review by raphaelspies — APPROVED** [review]

**Review by jeremysylee — APPROVED** [review]
nice work!

**Pull request merged**